webapp/security/helpers.py

```diff
@@ -100,3 +106,144 @@ def get_summarized_status(
             return friendly_names["not-affected"]
         elif status_count["DNE"] > 0:
             return friendly_names["DNE"]
+
+
+"""
```

Comment:
This has been refactored more clearly differentiate what is meant by "maintained" releases. These are the combination of lts releases (which are flagged as LTS in the UI), esm releases (which are also flagged as LST in the UI), and interim releases (which do not have a support tag). There aren't any interim releases right now so this can be qa'd as follows (on [landing](https://ubuntu-com-14270.demos.haus/security/cves)):
- "All maintained" should apply 24.04, 22.04, 20.04, 18.04, 16.04, and 14.04 filters when clicked
- "All LTS" should only apply 18.04, 16.04 and 14.04 filters when clicked